README.md

```diff
@@ -553,29 +591,39 @@ Now that the nodes have been initialized, we rotate the secrets files to overwri
 ```
   $docker secret create mldb_admin_password_v2 mldb_admin_password_v2.txt
 ```
+- Create mldb_wallet_password_v2.txt and use the following command to add a new docker secret for the wallet password:
```

Comment:
Create mldb_wallet_password_v2.txt and use the following command to add a new Docker secret for the wallet password:

(capitalize Docker)

Reply:
Fixed.